resources/zib/zib-wounds.WoundCharacteristics.xml

```diff
@@ -0,0 +1,286 @@
+    <name value="zib Wound-v3.3(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-burnwound-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/Burnwound-v3.4(2020EN)" />
+    <name value="zib Burnwound-v3.4(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="225552003" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:woundLength">
+      <path value="Observation.component" />
+      <sliceName value="woundLength" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:woundLength.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="39126-8" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:woundLength.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="WoundLength" />
+      <definition value="The length of the decubitus wound applies similarly to the definitions of Wound Depth and Wound Width." />
+      <alias value="Wondlengte" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <mapping>
+        <identity value="zib-pressureulcer-v3.4-2020EN" />
+        <map value="NL-CM:19.1.9" />
+        <comment value="WoundLength" />
+      </mapping>
+      <mapping>
+        <identity value="zib-wound-v3.3-2020EN" />
+        <map value="NL-CM:19.2.12" />
+        <comment value="WoundLength" />
+      </mapping>
+    </element>
+    <element id="Observation.component:woundWidth">
+      <path value="Observation.component" />
+      <sliceName value="woundWidth" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:woundWidth.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="39125-0" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:woundWidth.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="WoundWidth" />
+      <definition value="The width of the decubitus wound." />

```

Comment:
Change this to 'The width of the (decubitus) wound.'.

Reply:
fix